src/main/java/de/retest/recheck/persistence/NoGoldenMasterFoundException.java

```diff
@@ -1,17 +1,21 @@
 package de.retest.recheck.persistence;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
 public class NoGoldenMasterFoundException extends Exception {
 
 	private static final long serialVersionUID = 1L;
 
-	private final String filename;
+	private final List<String> filenames;
 
-	public NoGoldenMasterFoundException( final String filename ) {
-		super( "No Golden Master with the name '" + filename + "' has been found!" );
-		this.filename = filename;
+	public NoGoldenMasterFoundException( final String... filenames ) {
+		super( "No Golden Master with the following name(s) has/have been found: " + Arrays.toString( filenames ) );
```

Comment:
The `Arrays#toString(...)` variant was just to make it work. As described in https://github.com/retest/recheck.cli/issues/145, I thought about something like:

```
No Golden Master file(s) with the following name(s) found:
    foo
    bar
    baz
```

Have a look at [`Collectors#joining(...)`](https://www.concretepage.com/java/jdk-8/java-8-collectors-joining-example), this might be helpful here.